src/main/java/com/synopsys/integration/alert/workflow/startup/AlertStartupInitializer.java

```diff
@@ -117,11 +126,15 @@ private String convertKeyToPropery(final String descriptorName, final String key
     }
 
     private String getEnvironmentValue(final String propertyKey) {
+        String found = "No";
         String value = System.getProperty(propertyKey);
         if (StringUtils.isBlank(value)) {
-            logger.debug("Not found in system env, checking Spring env");
             value = environment.getProperty(propertyKey);
+            if (environment.containsProperty(propertyKey)) {
+                found = "Yes";
+            }
         }
+        logger.info("  {}: {}", propertyKey, found);
         return value;
```

Comment:
Should this return `Optional.ofNullable(value);`?

Reply:
I will change that